apps/cms/snapshot.yml

```diff
@@ -2633,16 +2633,26 @@ fields:
       interface: select-dropdown-m2o
       options:
         template: '{{slug}}'
+        filter:
+          _and:
+            - status:
+                _ncontains: published
       display: related-values
       display_options:
         template: '{{slug}}'
       readonly: false
       hidden: false
-      sort: null
-      width: half
+      sort: 11
+      width: full
       translations: null
       note: null
       conditions:
+        - name: Required if status is Published
+          rule:
+            _and:
+              - status:
+                  _contains: published
+          required: true
```

Comment:
Must set the Pack Template before publishing the NFT Template.

Reply:
Remove the above and make this field readonly instead. Users will need to set the Pack Template - NFT Template relationship via the Pack Template view.